venues/ICLR.cc/2018/Conference/python/programchairs/get-review-info.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+
+"""
+Output review information for all papers to a csv file.
+Paper Num, Title, Area Chair, reviews complete/total, Reviewer1, rating, confidence, Reviewer2...
+
+"""
+
+## Import statements
+import argparse
+from openreview import *
+import csv
+import sys
+sys.path.insert(0, "./..")
+import config
+
+## Handle the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+parser.add_argument('-f', '--file',help="specify an output file otherwise defaults to info.csv")
+parser.add_argument('--ac',help="specify area chair")
+args = parser.parse_args()
+if args.file == None:
+    args.file = 'info.csv'
+
+## Initialize the client library with username and password
+client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+reviews_total =0
+reviews_complete = 0
+# paper_data[paper_num]['title'] = title
```

Comment:
we can remove commented code

Reply:
This is meant to be a description of the paper_data structure. Since it is multiple layers I like to have it all written out in one place. Maybe there is a better way to put it so it doesn't look like unused code.